app/workers/hca/submission_job.rb

```diff
@@ -4,19 +4,23 @@ module HCA
   class SubmissionJob
     include Sidekiq::Worker
 
+    sidekiq_options unique_for: 30.minutes
```

Comment:
Good idea to add this?

Reply:
Ah, didn't realize the bug was related specifically to `scheduler`.

Reply:
Speaking of `sidekiq_options`, do we limit Sidekiq retries at all?  If not, potentially this job could fail for ~21 days before it gives up and sends the failure email.